### PR TITLE
Project anatomy DC-463

### PR DESCRIPTION
**Pull request by rasienko**
@sthielen @eltesoro-koji For your review, here's an update to the "projects" page. The primary goal of this PR is to address the Core package changes to the directory structure and configuration data (i.e., bye, bye .koji directory, hello koji.json file). So, please make sure it's accurate on that front. This page might be a good place to test out some of the terminology we were discussing earlier, too. So, let me know if it sparks any brainstorms on that front (and we can always update it again when we've decided on "final final" terms).

### CODE REVIEW

File: src/asciidoc/docs/develop/projects.adoc [hunk 1]
```diff
@@ -1,156 +1,144 @@
...
-The project environment includes a terminal connected to a running container with all the familiar tools, the same code editor that powers VSCode, and tools to help you deploy and publish your application to https://withkoji.com[withkoji.com].
+A repository on Koji contains everything that defines your development project, including the source code, assets, and configuration data.
+This topic describes a typical repository structure and defines the configuration data that is required to develop and deploy apps on Koji.
 
-== Project directory
+== Source control on Git
 
-The project directory at `/usr/src/app` includes a Git repository for your project with two remotes.
+The Koji platform uses Git repositories for source control.
+A directory on the Koji CDN stores the underlying files for each development project.
```
Comment (sthielen) [inline src/asciidoc/docs/develop/projects.adoc:12]: This feels potentially confusing - maybe more "Your repository is hosted on Koji's CDN"?

File: src/asciidoc/docs/develop/projects.adoc [hunk 2]
```diff
@@ -1,156 +1,144 @@
...
+When you start a new project by forking a scaffold or another app, the Koji platform creates a clone of the underlying repository.
+Your new project directory is associated with the following Git repositories.
```
Comment (sthielen) [inline src/asciidoc/docs/develop/projects.adoc:14]: `directory` feels odd here

**Review by sthielen — COMMENTED** [review]
added one small note - but otherwise looks good!

File: src/asciidoc/docs/develop/projects.adoc (hunk 1 above)
Comment (rasienko) [reply src/asciidoc/docs/develop/projects.adoc]: Awesome feedback! I made a change that I think addresses both comments. Let me know what you think!

**Review by eltesoro-koji — COMMENTED** [review]
For src/asciidoc/docs/develop/projects.adoc

BEFORE:     A repository on Koji contains everything that ...
SUGGESTION: A Koji repository contains everything that ...

BEFORE:     The Koji platform uses Git repositories for source control.
SUGGESTION: The Koji platform uses Git for source control.

ABOUT: A directory on the Koji CDN stores the underlying files for each development project.
COMMENT: This sentence doesn't seem to belong here. A CDN is not the same as a repository. Also, files *belong to* or *are associated with* a project. 

BEFORE:     In the following example, the app has directories for its backend and frontend services.
SUGGESTION: In the following example, the app's backend and frontend components have their own directories.

ABOUT: * `path` – Source code directory for the service.
QUESTION: Relative path from and including "frontend" or "backend" or "src"?

BEFORE: * `output` – Directory where your files will be compiled after the build commands are run.
AFTER:  * `output` – Directory where files produced by the build are stored.

BEFORE:     To open the settings in your app, you use the ...
SUGGESTION: To open the settings in your app, use the ...

BEFORE:     To implement these platform features in your app, you initialize the ...
SUGGESTION: To implement these platform features in your app, initialize the ...

**Comment by rasienko** [comment]
@eltesoro-koji Thanks! I made changes per your suggestions. Let me know if there's anything else.

**Review by sthielen — APPROVED** [review]
looks good to me!

**Pull request merged**